docs/database/sql.md

```diff
@@ -6,9 +6,34 @@ linkTitle: SQL
 
 **Status**: [Experimental][DocumentStatus]
 
-The SQL databases Semantic Conventions extend and override the [Database Semantic Conventions](database-spans.md)
-that describe common database operations attributes in addition to the Semantic Conventions
-described on this page.
+The SQL databases Semantic Conventions describes how common [Database Semantic Conventions](database-spans.md) apply to SQL databases.
+
+The following database systems (defined in the [`db.system`](./database-spans.md#notes-and-well-known-identifiers-for-dbsystem) set) are known to use SQL as their primary query language:
+
+- `cockroachdb`
```

Comment:
@jsuereth here's an example (tagging SQL databases) where enum tags would be great